sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -688,18 +678,19 @@
                 },
                 "supportsHttpsTrafficOnly": true,
                 "allowBlobPublicAccess": false,
-                "networkAcls": "[if(equals(parameters('storageAccountBehindVNet'), 'true'), variables('networkRuleSetBehindVNet'), json('null'))]",
-                "isHnsEnabled": "[equals(parameters('set_up_feature_store'), 'true')]",
-                "minimumTlsVersion": "TLS1_2"
+                "networkAcls": "[if(and(equals(parameters('storageAccountBehindVNet'), 'true'), equals(parameters('storageAccountOption'), 'new')), variables('networkRuleSetBehindVNet'), json('null'))]",
```

Comment:
I think networkAcls should apply when: (1) the storage is only for default blob; (2) the storage is for both default blob and offline store.

Reply:
yes, I think that's current behavior

Reply:
Current code:
when not equals(parameters('storageAccountOption'), 'new'))

the network ack is json('null').

But the original behavior is different.

Reply:
when not equals(parameters('storageAccountOption'), 'new')), this is feature store's offline store storage provision scenario, and it's a separate storage other than default storage. we should not apply networkAcls, which matches (2)